scripts/windows/build/rocksdb-win-arm32.ps1

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) Microsoft. All rights reserved.
```

Comment:
Seems this file can be merged into scripts\windows\setup\Install-Prerequisites.ps1

Reply:
Yes.